esmvalcore/preprocessor/_multimodel.py

```diff
@@ -206,6 +207,7 @@ def _combine(cubes):
         # https://scitools-iris.readthedocs.io/en/stable/userguide/
         #    merge_and_concat.html#common-issues-with-merge-and-concatenate
 
+        cube = remove_fx_variables(cube)
```

Comment:
I think that function updates the cube in place, so this should also work, right? Not sure if it makes a difference so feel free to leave it like this

```suggestion
        remove_fx_variables(cube)
```

Reply:
It has a `return` statetment because it's part of the default preprocessing chain. But I don't have a preference, I seems it works either way.

Reply:
I think it would be better to remove the `cube =`, because then it is immediately clear when reading the code that the function modifies the input cube.